specification/dimensions/pricingmodel.md

```diff
@@ -0,0 +1,39 @@
+# Pricing Model
+
+Pricing Model indicates the type of unit price applied to a charge at the time it was used or purchased. Pricing Model is commonly used to differentiate between charges at on-demand vs. reduced prices when investigating optimization opportunities, like for commitment-based discount coverage.
+
+The PricingModel column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the allowed values. PricingModel MUST be "Commitment-Based Discount" when CommitmentDiscountId is not null. PricingModel MUST be "Dynamic" when pricing is determined by the provider and may change over time, regardless of predetermined agreement pricing.
+
+## Column ID
+
+PricingModel
+
+## Display name
+
+Pricing Model
+
+## Description
+
+Type of unit price applied to a charge at the time it was used or purchased.
+
+## Content constraints
+
+|    Constraint   |      Value       |
+|:----------------|:-----------------|
+| Column required | True             |
+| Data type       | String           |
+| Allows nulls    | False             |
+| Value format    | list-of-values   |
+
+Allowed values:
+
+| Value                     | Description                                                                                                                                                                                                  |
+| :------------------------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Fixed                     | Charges priced at a constant rate, regardless of quantity used or purchased.                                                                                                                                 |
```

Comment:
How is "Fixed" different from "On-Demand"?
What would be an example?

Reply:
Here's how IBM Cloud defines in:
> _Fixed-rate pricing is based on an agreed upon monthly charge. If you buy an additional bare metal server, virtual server, or storage resource after the start of the monthly billing period, the cost of the resource for the first month is prorated based on the number of days remaining in the billing period. The prorated cost is billed in a separate invoice._
Thanks @dwbrown2 for sharing that!

I would vote we either make this more specific, like what IBM has, or just get rid of it and use `On-Demand` for the same purpose. Would love to know what others think.

Reply:
Thanks a lot, that clarifies.
BTW, IBM does not say what happens for the last month.

Thus, to support IBM Cloud, the value Fixed would make sense.
Maybe the description could be made more explicit, like "charges priced at a constant rate *over the billing period*".

Reply:
I'd vote to use On-Demand for that purpose

Reply:
Vote here to keep or remove: https://finopsfoundation.slack.com/archives/C053CNCRVCL/p1698650850102459

Reply:
It is removed